src/nvme/filters.c

```diff
@@ -133,8 +134,14 @@ int nvme_subsys_filter(const struct dirent *d)
 int nvme_scan_subsystems(struct dirent ***subsys)
 {
 	_cleanup_free_ char *dir = nvme_subsys_sysfs_dir();
+	int ret = scandir(dir, subsys, nvme_subsys_filter, alphasort);
 
-	return scandir(dir, subsys, nvme_subsys_filter, alphasort);
+	// If nvme_core isn't loaded, /sys/class/nvme-subsystem won't exist.
+	// Exit successfully with no NVMe subsystems.
```

Comment:
We don't have any `\\` comments in the code base as far I know. I'd rather keep it consistent and stay with the old style `/* */`.

Reply:
Sorry, not intended. Old habits die hard.